build.gradle

```diff
@@ -84,26 +74,24 @@ eclipse {
 
 eclipseClasspath.dependsOn("compileJavacc")
 
-task goloc(type: JavaExec, dependsOn: classes) {
+task goloc(type: JavaExec, dependsOn: [compileJava, processResources]) {
   main = goloCliMain
-  args = ['compile', '--output', goloClasses] + goloSources
-  classpath = sourceSets.main.runtimeClasspath
+  args = ['compile', '--output', sourceSets.main.output.classesDir] + goloSources
+  classpath = sourceSets.main.compileClasspath + files(compileJava.outputs) + files(processResources.outputs)
```

Comment:
goloc contributes to the runtimeClasspath, so it cannot depend on it at the same time. That would be a cyclic task dependency.